scripts/suse/apt/aptnotify

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+
+import os
+import hashlib
+
+CK_PATH = "/var/cache/salt/minion/rpmdb.cookie"
```

Comment:
I think this should be 
```suggestion
CK_PATH = "/var/cache/salt/minion/dpkg.cookie"
```

Reply:
You're right, this is a good point.

I think they decided to not changing this filename at this point because that would require refactors on the Java backend that generated the pkgset beacon configuration.

Currently, the configuration for the pkgset beacon is always defining `/var/cache/salt/minion/rpmdb.cookie` as the source for the cookie, regardless the OS.

Reply:
What about a generic name in that case? Could be named after the virtual pkg module: "pkg.cookie". Looking at the [Java code](https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/suse/manager/webui/services/pillar/MinionGeneralPillarGenerator.java#L47), the name should be easy to change.

Reply:
Hey, I agree we should have a generic name but I'm not sure what will happen after we change it.
The problem would be for existing costumer with everything already in place. We would force them to upgrade all the clients tools to generate the cookie file with the new name or beacon will not work after change the java code side.
Do we really want to figure out what will be all the side effects of changing it?

Reply:
Nothing special will happen if you change it. Just make sure you update minion *and* the backend.

Reply:
But we cannot guarantee that the custom update both at the same time. I am not sure what we do exactly on the java side, but maybe we can do it for both names, old and new. After some time we can remove the old name and only leave the generic name.

Reply:
Stupid proposal: symlink :-)

hey @isbm , glad to see you around!